Mapbox/MapboxMapsLocation/Pucks/LocationPuckManager.swift

```diff
@@ -20,15 +20,18 @@ internal enum PuckStyle {
 // MARK: PuckBackend
 /// This enum represents the different backends that can be used for Pucks
 public enum LocationPuck: Equatable {
-    case puck2D(customize: ((inout LocationIndicatorLayerViewModel) -> Void)? = nil) // Backed by `LocationIndicatorLayer`. Implement customize block to granularly modify the puck's styling.
-    case puck3D(customize: ((inout PuckModelLayerViewModel) -> Void))// Backed by `ModelLayer`. Implement customize block to granularly modify the puck's styling.
+    /// Backed by `LocationIndicatorLayer`. Optionally provide `LocationIndicatorViewModel` to granularly modify the puck's styling.
+    case puck2D(LocationIndicatorLayerViewModel? = nil)
```

Comment:
Interesting. I didn't know you could provide default values for an enum's associated value.

Reply:
Yep, it was added in Swift 5.3 I believe.